simpcli/Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 name = "simpcli"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
63b2f5ed799a160ccd1c2e83cd2b3dab6c0f3db8: This should be 0.3.0. We forgot to bump the version last time.